redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+            if guild.id not in self._server_mutes:
+                # initialize the guild in the cache to prevent keyerrors
+                self._server_mutes[guild.id] = {}
+            if after.id not in self._server_mutes[guild.id]:
+                try:
+                    await modlog.create_case(
+                        self.bot,
+                        guild,
+                        datetime.utcnow(),
+                        "smute",
+                        after,
+                        None,
+                        _("Manually applied mute role"),
+                    )
+                except RuntimeError as e:
+                    log.error(_("Error creating modlog case"), exc_info=e)
+                self._server_mutes[guild.id][after.id] = {
+                    "author": None,
+                    "member": after.id,
+                    "until": None,
+                }
+        if guild.id in self._server_mutes:
+            await self.config.guild(guild).muted_users.set(self._server_mutes[guild.id])
+
+    @commands.Cog.listener()
+    async def on_guild_channel_update(
+        self, before: discord.abc.GuildChannel, after: discord.abc.GuildChannel
+    ):
+        """
+        This handles manually removing overwrites for a user that has been muted
+        """
+        if after.id in self._channel_mutes:
+            before_perms: Dict[int, Dict[str, Optional[bool]]] = {
+                o.id: {name: attr for name, attr in p} for o, p in before.overwrites.items()
+            }
+            after_perms: Dict[int, Dict[str, Optional[bool]]] = {
+                o.id: {name: attr for name, attr in p} for o, p in after.overwrites.items()
+            }
+            to_del: List[int] = []
+            for user_id in self._channel_mutes[after.id].keys():
+                if user_id in before_perms and (
+                    user_id not in after_perms or after_perms[user_id]["send_messages"]
+                ):
+                    user = after.guild.get_member(user_id)
+                    if not user:
+                        user = discord.Object(id=user_id)
+                    log.debug(f"{user} - {type(user)}")
+                    to_del.append(user_id)
+                    try:
+                        log.debug("creating case")
+                        await modlog.create_case(
+                            self.bot,
+                            after.guild,
+                            datetime.utcnow(),
+                            "cunmute",
+                            user,
+                            None,
+                            _("Manually removed channel overwrites"),
+                            until=None,
+                            channel=after,
+                        )
+                        log.debug("created case")
+                    except RuntimeError as e:
+                        log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                        log.error("Error creating modlog case", exc_info=e)
```